tests/unit/test_conf.py

```diff
@@ -165,37 +166,37 @@ def test_pool_config_set_value():
 def test_pool_config_consume_and_then_consume_again():
 
     test_config = dict(test_pool_config)
-    with _pool_config_deprecations():
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", DeprecationWarning)
         consumed_pool_config = PoolConfig.consume(test_config)
+
     assert consumed_pool_config.encrypted is False
     consumed_pool_config.encrypted = "test"
 
     with pytest.raises(AttributeError):
-        _ = PoolConfig.consume(consumed_pool_config)
+        consumed_pool_config = PoolConfig.consume(consumed_pool_config)
 
-    with _pool_config_deprecations():
-        consumed_pool_config = PoolConfig.consume(
-            dict(consumed_pool_config.items())
-        )
-    with _pool_config_deprecations():
-        consumed_pool_config = PoolConfig.consume(
-            dict(consumed_pool_config.items())
-        )
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", DeprecationWarning)
+        consumed_pool_config = PoolConfig.consume(dict(consumed_pool_config.items()))
+        consumed_pool_config = PoolConfig.consume(dict(consumed_pool_config.items()))
```

Comment:
Why are we muting the `DeprecationWarning`s instead of asserting that they get emitted?

Reply:
Because in this line Deprecation warning was raised in this lines.

Reply:
Sure, but we expect them and probably want the tests to fail if they stop being emitted.